modules/likes/jetpack-likes-settings.php

```diff
@@ -598,8 +647,6 @@ function process_update_requests_if_sharedaddy_not_loaded() {
 	 * If sharedaddy is not loaded, we don't have the "Show buttons on" yet, so we need to add that since it affects likes too.
 	 */
 	function admin_settings_showbuttonon_init() {
-		?>
-		<?php
```

Comment:
Huh I guess a copy pasts from the code diff? https://github.com/Automattic/jetpack/pull/7175/files#diff-5a2eacc5a5ff9310feb3a0af5e364af4R575 cc @vindl